feature-group-definitions/container-style-queries.yml

```
@@ -1,4 +1,5 @@
 name: Container style queries
+description: CSS container style queries apply styles to an element based on the computed styles of its container.
```

One-off suggestion here, based on https://github.com/web-platform-dx/web-features/pull/801/files#r1554637063.

(Also avoiding starting the description with CSS. Not sure if that's too contorted.)

```suggestion
description: Container style queries with the `@container` at-rule apply styles to an element based on the values of custom properties of its container.
```